test/Bugs/randombug.baseline

```diff
@@ -63,10 +63,10 @@ Test case 30
 Passed
 Test case 31
 Error
-	at func2 (randombug.js:287:9)

```

Comment:
Consider removing the line numbers and columns from the baseline completely if this test isn't explicitly testing them.